mkosi/config.py

```diff
@@ -1436,6 +1437,9 @@ class Config:
     def name(self) -> str:
         return self.image_id or self.image or "default"
 
+    def machine_or_default(self) -> str:
```

Comment:
Maybe
```suggestion
    def machine_or_name(self) -> str:
```
since `_or_default` leaves open what the default is, whether it's `name()` or just `"default"`.